### PR TITLE
Product Updates - Release Notes

### PR DESCRIPTION
**Pull request by stef-cobaltio**
## Changelog

Release notes added for April, May, June, July and August of 2024.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Comment by mtsuji-cobalt** [comment]
@stef-cobaltio you'll need to add `.md` at the end of all these new pages:

Screenshot 2024-08-20 at 09 57 20

otherwise it turns out like this:

https://github.com/user-attachments/assets/5df527d3-2f43-4f61-be1e-24cd9a971824

**Comment by stef-cobaltio** [comment]
Thanks Maria - I just went in and updated all the links, please let me know if you see anything else.

File: content/en/Product Updates/release-notes-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+#### Cobalt Engagements: Introducing Digital Risk Assessments
+
+We are thrilled to introduce Cobalt Engagements, a new platform feature that streamlines the process of conducting Digital Risk Assessments (DRA) and empowers customers to take control of their offensive security testing like never before.
+
+Gone are the days of dealing with separate briefing documents, scoping questionnaires, and lengthy back-and-forth discussions to finalize a Statement of Work. With Cobalt Engagements, customers can now initiate a DRA engagement directly through our platform, simplifying and expediting the entire process.
+
+But that's not all - DRA is just the beginning. We are committed to continuously developing and introducing new engagements and functionality that will cater to our customers' evolving security needs. Stay tuned for more exciting updates in the future!![Severity Trend Chart](/release-notes/severity_trend.png "Severity Trend")
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-april-2024.md:30]: I think this Severity Trend image is from a previous post?

File: content/en/Product Updates/release-notes-april-2024.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- **Domain Protection:** Asset discovery is crucial for identifying dangling domains previously owned and used by an organization before an attacker exploits for malicious activities including subdomain takeovers. Identifying these dormant assets is essential to maintaining the integrity of an organization's online presence, ensuring all digital assets are accounted for and secured.
+- **Compliance and Regulation:** Different organizations/industries have different compliance requirements for various regulations and industry standards and must prove they have taken both offensive and defensive measures to manage digital risk. Regular assessments can help these organizations identify if sensitive information is inadvertently being exposed, which might lead to compliance issues.
+- **Post-Incident OSINT Analysis or Breach Intelligence Analysis:** After a breach, organizations may need to gather information about the attackers, their methods, and any additional threats they might face. This information can be valuable for understanding the scope of the breach, identifying the responsible parties, and preventing future incidents.
+---
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-april-2024.md:41]: Nitpick: don't think we need this extra divider line here and other pages

File: content/en/Product Updates/release-notes-july-2024.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+#### Attack Surface Updates
+
+In response to customer feedback and to better align with the comprehensive capabilities of the feature, we are excited to announce that 'Domains' has now been revamped as 'Attack Surface.' This new name reflects the broader scope and enhanced functionality that this feature offers to our users.
+
+As part of this update, we are introducing the all-new 'Attack Surface Insights' feature. We understand that managing multiple domains and hosts can be challenging for our customers, making it crucial to prioritize effectively. The new insights feature aims to simplify this process by providing high-level insights, enabling users to identify key focus areas and prioritize tasks seamlessly.
+
+With intuitive charts and visual aids, customers can now gain a consolidated overview of the health status of their attack surface, empowering them to make informed decisions and optimize their security strategies. This update underscores our commitment to helping our users achieve a more robust and proactive security posture.
+
+![Attack Surface Insights](/release-notes/Attack Surface Insights.png "Attack Surface Insights")
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-july-2024.md:24]: this image is not appearing. the spaces from the image file need to be removed or replaced with a character

File: content/en/Product Updates/release-notes-august-2024.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+In addition, we have also added a progress tracker to the scan details page. This tracker provides real-time insights into the duration of the scan and breaks down the progress into three key components: fingerprinting, crawling and scanning, also including error/warning messages when needed.
+
+These enhancements provide customers with immediate feedback on the progress of their scans, making it easier for them to identify any issues that may arise during the scanning process. 
+
+{{% pageinfo color="info" %}}
+View more scan controls available [here](https://docs.cobalt.io/platform-deep-dive/scans/scans/).
+{{% /pageinfo %}}
+
+![Scans Progress Tracker](/release-notes/Scan Progress.png "Scans Progress Tracker")
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-august-2024.md:28]: image file also needs to remove spaces in the name

**Comment by mtsuji-cobalt** [comment]
i noticed that the childpages are listed in alphabetical order here. i forget if you can rearrange the ordering on your end:
Screenshot 2024-08-21 at 08 39 15

**Comment by stef-cobaltio** [comment]
Thanks for the feedback, I've covered all the feedback. I tried adjusting the weight to reorder the child pages, this is the only change I am unsure of.

File: content/en/Product Updates/release-notes-july-2024.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Customers have expressed a strong desire to seamlessly integrate their DAST findings directly into their ticketing systems, a highly requested feature for DAST. In response to this feedback, we have enhanced our public API and Integration Builder to empower customers to:
+
+- Retrieve their DAST targets data
+- Access their DAST scans data
+- Retrieve their DAST findings data
+
+With these improvements, customers will now receive real-time notifications when a DAST finding is detected or resolved. Additionally, we have introduced recipe templates in the Integration Builder Library, allowing customers to effortlessly push their DAST findings to popular platforms such as Jira Cloud, GitHub Issues, and Azure DevOps Boards.
+
+## Steps to Integrate with Ticketing Systems
```
Comment (mtsuji-cobalt) [inline content/en/Product Updates/release-notes-july-2024.md:39]: sorry, last thing - i think this title could be reduced a bit in size. it's larger than the other titles on the page

Comment (stef-cobaltio) [reply content/en/Product Updates/release-notes-july-2024.md]: agree - went in and made it smaller (##### instead of ##)

**Review by mtsuji-cobalt — APPROVED** [review]

**Pull request merged**